Units/parser-puppetManifest.r/root-sep.d/expected.tags

```diff
@@ -0,0 +1,4 @@
+/tmp/classheir1	input.pp	/^   file { "\/tmp\/classheir1": ensure => file, mode => '0755' }$/;"	r	class:::base
+::base	input.pp	/^class base {$/;"	c
+::base./tmp/classheir1	input.pp	/^   file { "\/tmp\/classheir1": ensure => file, mode => '0755' }$/;"	r	class:::base
```

Comment:
Where is this `::base./tmp/classheir1` coming ? 
I do not think you can write this in puppet language. So one could not jump to this tag.  
Maybe we could omit this ?

Reply:
`.` is ctags global built-in separator.
See the document I wrote.

Tell me all the combinations of kinds you don't expect. I will consider to extend command line interface for filtering out them.
If you need just replace the chars of separators, read the document. It tells how to do it.